plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -2256,6 +2281,8 @@ instance Flat DefaultFun where
               go 89 = pure LengthOfArray
               go 90 = pure ListToArray
               go 91 = pure IndexArray
+              go 94 = pure InsertCoin
```

Comment:
Why the gap between 94 and 97?  Are we expecting it to be filled by some more Value-related functions?

Reply:
Yes. I will add them in the order written in the CIP.